plugins/home/src/api/LocalStorageVisitsApi.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { IdentityApi } from '@backstage/core-plugin-api';
+import { Visit } from './VisitsApi';
+import { VisitsApiFactory } from './VisitsApiFactory';
+
+/** @public */
+export type LocalStorageVisitsApiOptions = {
+  limit?: number;
+  identityApi: IdentityApi;
+};
+
+/**
+ * @public
+ * This is a reference implementation of VisitsApi using window.localStorage.
+ */
+export class LocalStorageVisitsApi extends VisitsApiFactory {
+  private readonly localStorage = window.localStorage;
```

Comment:
Sticking to that naming convention this would change to VisitsLocalStorageApi as discussed.

Reply:
addressed in 29662110254b1e8be15dc7a739eddf005f0d9f5a